ccmlib/scylla_node.py

```diff
@@ -577,8 +576,9 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
 
         # The '--kernel-page-cache' was introduced by
         # https://github.com/scylladb/scylla/commit/8785dd62cb740522d80eb12f8272081f85be9b7e from 4.5 version
-        current_node_version = self.node_install_dir_version
-        if current_node_version and LooseVersion('666.development') < LooseVersion(current_node_version) >= LooseVersion('4.5'):
+        major_node_scylla_version = self.get_release_major_scylla_version()
+
+        if major_node_scylla_version and float(major_node_scylla_version) >= 4.5:
```

Comment:
If you add this get_release_major_scylla_version() function, I suggest you simplify its usage by ensuring it always returns a float, then you can just have "if major_node_scylla_version >= 4.5".

However, see also below my question about enterprise version, which I'm not sure how this code (or the previous code it replaces) handles....